docs/README.md

```diff
@@ -326,25 +326,26 @@ There are two methods for it:
 - setInterval allows us to run a function repeatedly, starting after the interval of time, then repeating continuously at that interval.
 - Infinities
```

Comment:
Add it as a new heading
Also add this topic in the index (at the top)

Reply:
Done

Reply:
I added this as a new Heading after "DOM Manipulation".